### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by benjaminjb**
Screen Shot 2023-10-26 at 10 30 32 AM
Screen Shot 2023-10-26 at 10 30 14 AM
Screen Shot 2023-10-26 at 10 00 51 AM

### CODE REVIEW

File: docs/content/reference/pgo_show_backup.md [hunk 1]
```diff
@@ -8,28 +8,46 @@ Show backup information for a PostgresCluster
 ### Synopsis
 
 Show backup information for a PostgresCluster from 'pgbackrest info' command.
-
-#### RBAC Requirements
+
```
Comment (tony-landreth) [inline docs/content/reference/pgo_show_backup.md:11]: Is that a tab?

Comment (benjaminjb) [reply docs/content/reference/pgo_show_backup.md]: Yeah, I removed the tab from the code, but didn't regen the docs, thx!

**Review by tony-landreth — APPROVED** [review]

File: internal/cmd/version.go [hunk 2]
```diff
@@ -33,17 +33,26 @@ func newVersionCommand(config *internal.Config) *cobra.Command {
...
 	}
 
 	// No arguments for 'version'
 	cmd.Args = cobra.NoArgs
 
+	cmd.Example = internal.FormatExample(`# Request the version of the client and the operator
+pgo version
+
+### Example output
+Client Version: v0.3.0
```
Comment (tjmoore4) [inline internal/cmd/version.go:53]: :thinking: When should this value be updated?

File: docs/content/reference/pgo_version.md [hunk 3]
```diff
@@ -9,17 +9,32 @@ PGO client and operator versions
...
+### Examples
+
+```
+# Request the version of the client and the operator
+pgo version
+
+```
+### Example output
+```
+Client Version: v0.3.0
```
Comment (tjmoore4) [inline docs/content/reference/pgo_version.md:34]: :thinking: Are these values what we want here?

**Review by tjmoore4 — APPROVED** [review]

File: internal/cmd/version.go (hunk 2 above)
Comment (benjaminjb) [reply internal/cmd/version.go:53]: 👍 I'm going to make a ticket for bumping the version / releasing.

File: docs/content/reference/pgo_version.md (hunk 3 above)
Comment (benjaminjb) [reply docs/content/reference/pgo_version.md:34]: Changing the operator version at least to something _more likely_ to be seen.

**Pull request merged**